framework/Source/GPUImageMovie.m

```diff
@@ -695,6 +695,14 @@ - (void)endProcessing;
         [currentTarget endProcessing];
     }
     
+    // dgm: added this - otherwise we were getting crashes when the playerItemOutput
+    // was processing already scheduled work after it had supposedly been stopped.
+    if (playerItemOutput && self.playerItem) {
+        [self.playerItem removeOutput:playerItemOutput];
```

Comment:
This doesn't cause issues if the output has already been removed?  If you call this twice does it crash?